scenes/quests/story_quests/stella_and_the_missing_star/0_intro/intro.tscn

```diff
@@ -0,0 +1,122 @@
+[gd_scene load_steps=10 format=4 uid="uid://c1gdct760l86h"]
+
+[ext_resource type="TileSet" uid="uid://07fq3rspk8ia" path="res://scenes/tileset.tres" id="1_2qnws"]
+[ext_resource type="Script" uid="uid://x1mxt6bmei2o" path="res://scenes/menus/intro/components/intro.gd" id="1_o3x1t"]
+[ext_resource type="PackedScene" uid="uid://iu2q66clupc6" path="res://scenes/game_elements/characters/player/player.tscn" id="2_17a4c"]
+[ext_resource type="Resource" uid="uid://cqjngitoc5yx7" path="res://scenes/quests/story_quests/stella_and_the_missing_star/0_intro/stella_intro.dialogue" id="2_jlqtq"]
+[ext_resource type="Texture2D" uid="uid://yrinqbpqojoa" path="res://scenes/quests/story_quests/stella_and_the_missing_star/assets/constellation-frames.png" id="4_blo5s"]
+
+[sub_resource type="Animation" id="Animation_ld06i"]
+length = 0.001
+tracks/0/type = "value"
+tracks/0/imported = false
+tracks/0/enabled = true
+tracks/0/path = NodePath("Sky/Constellation:frame")
+tracks/0/interp = 1
+tracks/0/loop_wrap = true
+tracks/0/keys = {
+"times": PackedFloat32Array(0),
+"transitions": PackedFloat32Array(1),
+"update": 1,
+"values": [0]
+}
+tracks/1/type = "value"
+tracks/1/imported = false
+tracks/1/enabled = true
+tracks/1/path = NodePath("OnTheGround/Player:position")
+tracks/1/interp = 1
+tracks/1/loop_wrap = true
+tracks/1/keys = {
+"times": PackedFloat32Array(0),
+"transitions": PackedFloat32Array(1),
+"update": 0,
+"values": [Vector2(153, 366)]
+}
+
+[sub_resource type="Animation" id="Animation_7swpb"]
+resource_name = "walk_off_screen"
+length = 2.0
+tracks/0/type = "value"
+tracks/0/imported = false
+tracks/0/enabled = true
+tracks/0/path = NodePath("OnTheGround/Player:position")
+tracks/0/interp = 1
+tracks/0/loop_wrap = true
+tracks/0/keys = {
+"times": PackedFloat32Array(0, 2),
+"transitions": PackedFloat32Array(1, 1),
+"update": 0,
+"values": [Vector2(153, 366), Vector2(1010, 366)]
+}
```

Comment:
TODO: This should instead play the walking animation and make the player walk at the standard speed...